integrations/pinecone/pyproject.toml

```
@@ -25,7 +25,8 @@ classifiers = [
 ]
 dependencies = [
   "haystack-ai>=2.11.0",
-  "pinecone>=3", # our implementation is not compatible with pinecone <3
+  "pinecone>=3", # our implementation is not compatible with pinecone <3,
+  "pinecone[asyncio]>=3",
```

```suggestion
  "pinecone[asyncio]>=3", # our implementation is not compatible with pinecone <3,
```
this should work, since `asyncio` is just an extra group of dependencies
https://github.com/pinecone-io/pinecone-python-client/blob/4e3ef2ef255b41bbb5273bcdc4d7a32cd0bdb1b0/pyproject.toml#L101

---

I mean just specifying `"pinecone[asyncio]>=3",` and removing `"pinecone>=3",`

---

I did update it. I think I missed it in the commit.